frontend/src/pages/Students.vue

```diff
@@ -2,294 +2,290 @@
+        </b-table>
       </section>
     </div>
-  </Page>      
+  </Page>
 </template>
 
-
 <script>
-
-
 const API_URL = "/api/students";
 
-import PageHeader from '../components/PageHeader.vue'
-import Page from '../components/Page.vue'
-
+import PageHeader from "../components/PageHeader.vue";
+import Page from "../components/Page.vue";
 
 export default {
-        data() {
-            return {
-                data: [
-                ],
-                selected: null,
-                sortIcon: 'arrow-up',
-                sortIconSize: 'is-small',
-                sortDirection: 'asc',
-                columns: [
-                    {
-                        field: 'CreatedAt',
-                        label: 'Date Joined',
-                        searchable: true,
-                    },
-                    {
-                        field: 'StudentID',
-                        label: 'Student ID',
-                        searchable: true,
-                    },
-                    {
-                        field: 'FullName',
-                        label: 'Student Name',
-                        searchable: true,
-                    },                               
-                    {
-                        field: 'StatusID',
-                        label: 'Status',
-                        searchable: true,
-                    },
-                    {
-                        field: 'PhoneNumber',
-                        label: 'Phone Number',
-                        searchable: true,
-                    }
-                ]
-            }
+  data() {
+    return {
+      selected: null,
+      sortIcon: "arrow-up",
+      sortIconSize: "is-small",
+      sortDirection: "asc",
+      columns: [
+        {
+          field: "CreatedAt",
+          label: "Date Joined",
+          searchable: true,
+        },
+        {
+          field: "StudentID",
+          label: "Student ID",
+          searchable: true,
+        },
+        {
+          field: "FullName",
+          label: "Student Name",
+          searchable: true,
+        },
+        {
+          field: "Status",
+          label: "Status",
+          searchable: true,
         },
-        computed:{
-          tableData(){
-            return this.data.map(student => {
-              return {
-                StudentID: `${student.StudentID}`,
-                FullName: `${student.FirstName} ${student.LastName}`,
-                CreatedAt: `${student.created_at}`,
-                StatusID: `${student.StatusID}`,
-                PhoneNumber: `${student.PhoneNumber}`  
-              }                
-            })
-          }
+        {
+          field: "PhoneNumber",
+          label: "Phone Number",
+          searchable: true,
         },
-        components:{
-          Page,
-          PageHeader
-        },        
+      ],
+    };
+  },
+  computed: {
+    tableData() {
+      return this.$store.getters.students.map((student) => {
```

Comment:
This can be refactored out using the vuex `mapGetters` helper https://vuex.vuejs.org/guide/getters.html#the-mapgetters-helper

Reply:
Thanks! have made the changes